src/server/createMockServer.ts

```diff
@@ -10,7 +10,7 @@ import extractServiceDefinition from './extractServiceDefinition';
 import getRandomInt from '../util/getRandomInt';
 import delay from '../util/delay';
 import { ILogger } from '../log/logger.interface';
-
+import { isEqual } from 'lodash';
```

Comment:
As I mentioned, I'd rather not use `lodash`, and write my own method. 
I can do it later tonight, for sake of merging this, can you just declare and call an empty isEqual function?

Reply:
Sure makes sense. You could also do some wildcard comparison? So one parameter could be a wildcard, so it would compare the value